includes/Core/Authentication/Authentication.php

```diff
@@ -85,13 +85,13 @@ final class Authentication {
 	private $user_input_state = null;
 
 	/**
-	 * User_Input_Settings
+	 * User_Input
 	 *
 	 * @since 1.20.0
```

Comment:
This should probably increment here as the property is different.

Reply:
Updated, thank you!